doc/attr.xml

```diff
@@ -1310,3 +1310,42 @@ fail]]></Example>
   </Description>
 </ManSection>
 <#/GAPDoc>
+  
+<#GAPDoc Label="HamiltonianPath">
+<ManSection>
+  <Prop Name="HamiltonianPath" Arg="digraph"/>
```

Comment:
This shouldn't be a `Prop` - the type should agree with how `HamiltonianPath` (currently it's an Operation, and so it should say `Oper` here, although further down I suggest that `HamiltonianPath` should instead be declared as an attribute, and so this should be `Attr`).